### PR TITLE
feat: added box with link to enterprise in header

### PR DESCRIPTION
**Pull request by bartsmykla**
- Missing proper message
- Missing proper link

image

### CODE REVIEW

**Comment by bloqhead** [comment]
- I've updated the `EnterpriseBox` accompanying button text to "Do more with Kuma →"
- Revised the button to link to https://kuma.io/enterprise (in a new tab/window)

File: src/components/Utils/EnterpriseBox.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <KAlert
+    class="py-1"
+    appearance="success"
+    size="small"
+  >
+    <template slot="alertMessage">
+      <div class="alert-content flex items-center text-sm">
+        <div class="mr-3">
+          Do more with Kuma <span class="alert-content__arrow">&rarr;</span>
```
Comment (bloqhead) [inline src/components/Utils/EnterpriseBox.vue:10]: @bartsmykla I've updated this text based on Marco's feedback.

File: src/components/Utils/EnterpriseBox.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    </template>
+  </KAlert>
+</template>
+
+<script>
+export default {
+  name: 'EnterpriseBox',
+  data () {
+    return {
+      url: 'https://kuma.io/enterprise/'
```
Comment (bloqhead) [inline src/components/Utils/EnterpriseBox.vue:33]: @bartsmykla Changed this to the [Kuma Enterprise page](https://kuma.io/enterprise/).

**Review by bloqhead — APPROVED** [review]

**Comment by subnetmarco** [comment]
Can we merge this @nickolaev ? @bloqhead has updated it and it's good to go.

**Pull request merged**